src/schemas/transfer-benefits/schema.js

```diff
@@ -0,0 +1,105 @@
+import definitions from '../../common/definitions';
+import _ from 'lodash';
+
+export default {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'APPLICATION FOR FAMILY MEMBER TO USE TRANSFERRED BENEFITS',
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions,
+    'privacyAgreementAccepted',
+    'ssn',
+    'gender',
+    'date',
+    'fullName',
+    'address',
+    'phone',
+    'bankAccount',
+    'educationType',
+    'school',
+    'postHighSchoolTrainings',
+    'dateRange',
+    'nonMilitaryJobs'
+  ),
+  properties: {
+    privacyAgreementAccepted: {
+      $ref: '#/definitions/privacyAgreementAccepted'
```

Comment:
I'm starting to dislike `$ref`, since it means we have two different ways of sharing definitions. It doesn't seem like it buys us much, given that we're already importing shared definitions and could easily just set fields to the object directly:

```
privacyAgreementAccepted: definitions.privacyAgreementAccepted
```

`$ref` does make the schema size a bit smaller, but outside of the address definitions (which has countries and states listed), it doesn't seem like much of a difference.

I'm not saying we should change anything right now, more just curious about your thoughts @lihanli @markolson.